src/main/java/sparkles/util/Ui.java

```diff
@@ -0,0 +1,61 @@
+package sparkles.util;
+
+import java.util.Scanner;
+
+/**
+ * A Ui object that deals with user interactions.
+ */
+public class Ui {
+
+    private static final String DASH = "     ____________________________________________________________";
+    private final Scanner sc;
+
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Prints the welcome message of Sparkles.
+     */
+    public void showWelcome() {
+        showLine();
+        print("     *Hello, I am Sparkles*\n\n     How can I help you?");
+        showLine();
+    }
+
+    public void showLine() {
+        print(DASH);
+    }
+
+    public void print(String str) {
+        System.out.println(str);
+    }
+
+    /**
+     * Print the number of task in the list now.
+     *
+     * @param i number of task
+     */
+    public void printListSize(int i) {
```

Comment:
Maybe you can name it showListSize instead? Since all the other methods are named showLine/ showLoadingError

Reply:
Noted with thanks for the all the method names pointed out!